.github/workflows/ci.yml

```
@@ -1,148 +1,172 @@
-        run: npm ci
-        working-directory: server
-
-      - name: Generate Prisma client
-        run: npx prisma generate --schema=src/database/prisma/schema
-        working-directory: server
-
-      - name: Build server
-        run: npm run build
-        working-directory: server
\ No newline at end of file
+name: CI
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)